tests/test_handwritten.py

```diff
@@ -0,0 +1,104 @@
+import unittest, os
+from nelox.parser import Parser
+from nelox.scanner import Scanner
+
+class TestHandWritten(unittest.TestCase):
+    def test_case1(self):
+        base = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+        path = os.path.join(base, "dataset", "handwritten", "testcase1.txt")
+        with open(path, "r", encoding="utf-8") as f:
+            src = f.read()
+        parser = Parser(Scanner(src))
+        self.assertIsInstance(parser.parse(), list)
```

Comment:
This doesn't really test anything, parse will return a list always.  Perhaps for now it's enough to just check that parse doesn't throw an exception. (I forgot we don't have an interpreter yet)